cylc/flow/scheduler_cli.py

```diff
@@ -147,7 +147,6 @@ def get_option_parser(add_std_opts=False):
             "Set the final cycle point. "
             "This command line option overrides the workflow "
             "config option '[scheduling]final cycle point'. "
-            "Use a value of 'reload' to reload from flow.cylc in a restart."
```

Comment:
Removed this as it implies changes to flow.cylc will not be picked up normally; not the case; see https://github.com/cylc/cylc-doc/pull/459